pandas/core/dtypes/dtypes.py

```diff
@@ -5,15 +5,15 @@
 from pandas import compat
 from pandas.core.dtypes.generic import ABCIndexClass, ABCCategoricalIndex
 
+from .base import ExtensionDtype
 
-class ExtensionDtype(object):
+
+class PandasExtensionDtype(ExtensionDtype):
```

Comment:
why would this not be just ExtensionDtype

Reply:
We define methods like `__repr__`, caching, etc. that are not part of the interface.

Reply:
__repr__ is reasonable to be part of the interface, caching I suppose is ok here